packages/frontend/scss/message/_message.scss

```diff
@@ -98,6 +98,29 @@ $info-text-max-width: 400px;
           cursor: pointer;
         }
       }
+      &.call {
+        > .text {
+          margin-inline-end: 24px;
+          white-space: nowrap;
```

Comment:
Do these always fit on narrow screens? And for all locales?

Reply:
I can't test it for all languages but the longest possible text is "Outgoing call <1 minute" which means 3 words it should be sufficient.

I did not apply different styles for rtl languages since it's more a question of taste wether the icon should be before he text or after